### DISCUSSION TITLE
How to debug on production? :))

### DISCUSSION THREAD

**Discussion by ghost** [spatie/ray#923] [Q&A]
Might be a dumb question with obvious answer, but still wanted to ask.

You can't change the env or set debug to true... So how do you do it ?

PS: I did find this topic [https://github.com/spatie/ray/discussions/47](url), but it's from 2021, so I was wondering if there is update now 3 years later? :)

**Comment by slaFFik** [comment]
Same issue, and there is no update :(

**Comment by ghost** [comment]
Seems the discussion from the link was deleted 🥲

However, you can edit the ray config and set IP condition when to enable ray debugging as a workaround.
While not a great solution, it's better than enabling debug in the env.

**Reply by slaFFik** [reply]
I assume you are talking about laravel-ray integration?
I'm not using Laravel :(
